transit_odp/timetables/csv.py

```diff
@@ -115,8 +115,8 @@
             "last modified' date is sooner than 'Effective stale date due to "
             "end date' (if present) and today’s date from which the file is "
             "created equals or passes 'Effective stale date due to effective "
-            "last modified date' and Last modified date < OTC Effective start "
-            "date - FALSE. </br></br>"
+            "last modified date' and Last modified date < 'Effective stale date "
+            "due to OTC effective date' = FALSE. </br></br>"
```

Comment:
Could you please make minor changes to Stale - OTC Variation please, to keep it in the same format as others. Please change  If Last modified date < 'Effective stale date due to OTC effective date' **=** TRUE. 
And remove the extra than in "greater than or equal to **than**"

Reply:
Fixed this